src/main/java/com/openshift/jenkins/plugins/pipeline/OcAction.java

```diff
@@ -63,14 +64,18 @@ public OcAction(String server, String project, boolean skipTLSVerify, String caP
         public int status;
         @Whitelisted
         public HashMap<String, String> reference = new HashMap<String, String>();
+        @Whitelisted
+        public boolean verbose = false;
```

Comment:
should this flag be propagated to all the commands in the GroovyDSL?
if so, we can do it in a separate PR.

Reply:
I forwarded it from the Step verbose property, to ensure that OcActionResult doesn't print the entire yaml/json on Jenkins. 
But I might misunderstand the question and would love suggestions on how to improve it.